packages/controllers/src/workers/WorkerController.ts

```diff
@@ -96,8 +93,11 @@ export class WorkerController extends SafeEventEmitter {
     }
 
     console.log('Parent: Sending Command', message);
-
-    return await workerObj.commandEngine.command(message, timeout);
+    const response: any = await workerObj.rpcEngine.handle(message);
```

Comment:
We should be able to use a more specific type here, probably `JsonRpcResponse<unknown>`.